src/v/pandaproxy/schema_registry/protobuf.cc

```diff
@@ -438,7 +438,7 @@ ss::future<pb::FileDescriptorProto> build_file_with_refs(
             continue;
         }
         auto dep = co_await store.get_subject_schema(
-          ref.sub, ref.version, include_deleted::no);
+          ref.sub, ref.version, include_deleted::yes);
```

Comment:
This is also inconsistent with json, as we don't throw a `no_reference_found_for`